.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+jobs:
+  tests:
+    docker:
+      # Primary docker container where the tests will be run
+      - image: ubuntu:18.04
+
+      # Secondary docker container for database service
+      - image: postgres:10.6
+        environment:
+          POSTGRES_USER: circleci
+
+    working_directory: ~/hetmech-backend
+
+    steps:
+      - checkout
+      - run:
+          name: Install miniconda
+          working_directory: ~/
+          command: |
+            apt update && apt upgrade -y
```

Comment:
is `--yes` or `--assume-yes` not valid for apt? The repo policy is to always use long arguments when possible.